test/images/image-util.sh

```diff
@@ -117,23 +141,46 @@ push() {
   docker_version_check
   TAG=$(<"${IMAGE}"/VERSION)
   if [[ -f ${IMAGE}/BASEIMAGE ]]; then
-    archs=$(listArchs)
+    base_images=$(listBaseImages)
+    # NOTE(claudiub): if the REMOTE_DOCKER_URL var is not set, or it is an empty string, we must skip
+    # pushing the Windows image and including it into the manifest list.
+    if [[ ! -v "REMOTE_DOCKER_URL" || -z "${REMOTE_DOCKER_URL}" ]]; then
```

Comment:
Dumb question: does the `!` only apply to the first half of the `... || ...` or to the entire expression?